typescript/src/app.ts

```diff
@@ -0,0 +1,48 @@
+import {
+  authenticate,
+  createOrder,
+  getAccountInfo,
+  getOpenOrders,
+  listAvailableMarkets,
+  onboardUser,
+} from "./utils/api";
+import { Account, SystemConfig } from "./utils/types";
+import { shortString } from "starknet";
+
+// Example usage of the Paradex API
+(async () => {
+  // TODO: Get from /system/config
+  const apiBaseUrl = "https://api.testnet.paradex.trade/v1";
+  const chainId = shortString.encodeShortString("PRIVATE_SN_POTC_SEPOLIA");
+  const config: SystemConfig = {
+    apiBaseUrl,
+    starknet: { chainId },
+  };
+
+  // TODO: Add key derivation
+  const account: Account = {
+    address: process.env.ACCOUNT_ADDRESS || "",
+    publicKey: process.env.PUBLIC_KEY || "",
+    privateKey: process.env.PRIVATE_KEY || "",
+    ethereumAccount: process.env.ETHEREUM_ACCOUNT || "",
+  };
+
+  try {
+    await onboardUser(config, account);
+    account.jwtToken = await authenticate(config, account);
```

Comment:
We may also provide an example of refreshing the auth token every 3-4 mins as the token is short-lived;
But probably for the simplicity of the example, it is better to keep as-is;

Reply:
I'll add a follow-up task